src/ray/raylet/worker_pool.cc

```diff
@@ -217,6 +223,15 @@ Process WorkerPool::StartWorkerProcess(
       }
       continue;
     }
+    auto node_manager_port_position = token.find(kNodeManagerPortPlaceholder);
+    if (node_manager_port_position != std::string::npos) {
+      auto replaced_token = token;
+      replaced_token.replace(node_manager_port_position,
+                             strlen(kNodeManagerPortPlaceholder),
+                             std::to_string(node_manager_port_));
+      worker_command_args.push_back(replaced_token);
+      continue;
+    }
 
```

Comment:
Will this method always be called after SetNodeManagerPort?

Reply:
Currently yes. But good catch! I should add `RAY_CHECK(node_manager_port_ != 0)` here.

Reply:
Thanks for adding that!